MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/SymbolLocationLayerRenderer.java

```diff
@@ -343,7 +343,7 @@ private void refreshSource() {
 
     GeoJsonSource source = style.getSourceAs(LOCATION_SOURCE);
     if (source != null) {
-      locationSource.setGeoJson(locationFeature);
+      locationSource.setGeoJson(Feature.fromJson(locationFeature.toJson()));
```

Comment:
`locationFeature.toJson()` should be sufficient.

Reply:
done